src/tests/sign-up-form-negative-e2e.spec.js

```diff
@@ -0,0 +1,86 @@
+
+import { test, expect } from '@playwright/test';
+import { SignUpPage } from "../pages/SignUpPage";
+
+const redBorderColor = 'rgb(220, 53, 69)';
+
+test.describe.only("Sign-up form validation", ()=> {
+    let page;
+    let signUpPage;
+
+    test.beforeAll(async ({browser})=>{
+        page = await browser.newPage();
+
+        signUpPage = new SignUpPage(page);
+
+        await signUpPage.navigate();
+    });
+
+    test.beforeEach(async () => {
+        await signUpPage.openSignUpWindow();
+    });
+
+    test.afterEach(async () => {
+        await signUpPage.closeSignUpWindow();
+    });
+
+    test("T-01 verify the name empty and red border", async ()=> {
+        await signUpPage.userNameInput.click();
+        await signUpPage.focusOutFromField();
+
+        const firstErrorText = await signUpPage.getWarningNotification(0);
+
+        const borderColor = await signUpPage.getFieldBorderColor(signUpPage.userNameInput);
+
+        expect(firstErrorText).toStrictEqual('Name required');
+        expect(borderColor).toBe(redBorderColor);
```

Comment:
use web first assertions
https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-text
https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-css

Reply:
Consider the option to move it to page object